docs/source/api_ref_training.rst

```diff
@@ -76,6 +76,20 @@ Utilities to reduce memory consumption during training.
 
 .. _metric_logging_label:
 
+Schedulers
+----------
+
+Utilities to control lr during the training process.
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    get_cosine_schedule_with_warmup
+
+.. _lr_scheduler_label:
```

Comment:
Is this right? Seems to me like lr scheduler label and metric logging label need to be swapped

Reply:
Are you speaking about 2 sections?

Reply:
I seen now what was your idea, fixed!